document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/service/impl/AccountsServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.document.generator.accounts.service.impl;
+
+import static uk.gov.companieshouse.document.generator.accounts.AccountsDocumentInfoServiceImpl.MODULE_NAME_SPACE;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.accounts.Accounts;
+import uk.gov.companieshouse.document.generator.accounts.data.accounts.AccountsManager;
+import uk.gov.companieshouse.document.generator.accounts.service.AccountsService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Service
+public class AccountsServiceImpl implements AccountsService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MODULE_NAME_SPACE);
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Accounts getAccounts(String resource) {
+        LOG.info("Getting accounts data");
+
+        ResponseEntity<Accounts> accounts = AccountsManager.getAccounts(resource);
```

Comment:
The private-sdk won't be returning a ResponseEntity. See get() in [AccountResourceHandler](https://github.com/companieshouse/api-sdk-java/blob/master/src/main/java/uk/gov/companieshouse/api/handler/transaction/account/AccountResourceHandler.java). The AccountsManager should return the Accounts object or throw a custom exception to represent the ApiErrorResponseException thrown by the SDK.

If you remove the static from the methods in the AccountsManager it can then be mocked for unit testing which means the unit tests can be used with minimal modification (only need to replace the mock) when the private SDK is integrated.